map/manage-operational-layers/src/main/java/com/esri/samples/manage_operational_layers/ManageOperationalLayersSample.java

```diff
@@ -64,8 +65,12 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
       // create an ArcGISMap with the topographic basemap
```

Comment:
Just noticing most of the other comments for this line in other samples now say `create a map with..`

Might be nice to make it uniform?

So as you have in the next file:
```suggestion
      // create a map with the topographic basemap style
```

Reply:
Thanks yes that was the plan to make it consistent, so will tidy this one!